hca/dss/__init__.py

```diff
@@ -213,15 +216,16 @@ def _download_file(self,
                 logger.info("%s", "File {}: GET SUCCEEDED. Stored at {}.".format(file_uuid, dest_path))
 
     @classmethod
-    def _file_path(cls, checksum):
+    def _file_path(cls, checksum, version='v2'):
         """
         returns a file's relative local path based on the nesting parameters and the files hash
         :param checksum: a string checksum
         :return: relative Path object
         """
-        assert(sum(cls.DIRECTORY_NAME_LENGTHS) <= len(checksum))
+        file_prefix = '_'.join(['files'] + list(map(str, cls.DIRECTORY_NAME_LENGTHS)))
+        path_pieces = [DOWNLOAD_DIR_ROOT, version, file_prefix]
```

Comment:
```suggestion
        path_pieces = ['.hca', 'v2', file_prefix]
```

Keep all the bits in one place.

Reply:
Alright I'll make these changes.